### PR TITLE
Cross build with Scala 2.10, 2.11

### PR DESCRIPTION
**Pull request by gianm**
- Remove Scala 2.9.1 build
- Update Curator to 2.6.0
- Update to fork of scala-util [see https://github.com/metamx/scala-util/pull/10]
- Fix logging for tests

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -41,19 +41,13 @@ releaseSettings
...
-
-def ScalatraCross = CrossVersion.binaryMapped {
-  case "2.9.1" => "2.9.2"
-  case x => x
-}
+val curatorVersion = "2.6.0"
 
 libraryDependencies ++= Seq(
-  "com.metamx" %% "scala-util" % "1.9.0",
+  "com.metamx" %% "scala-util" % "1.11.0-SNAPSHOT",
```
Comment (xvrl) [inline build.sbt]: I think we just need to cut a release and change this before I can merge this.

File: build.sbt [hunk 2]
```diff
@@ -62,17 +56,31 @@ libraryDependencies ++= Seq(
...
+  "junit" % "junit" % "4.11" % "test",
+  "com.novocode" % "junit-interface" % "0.10" % "test",
   "org.apache.derby" % "derby" % "10.10.1.1" % "test",
   "org.apache.curator" % "curator-test" % curatorVersion % "test",
-  "org.scalatra" % "scalatra-test" % "2.2.1" % "test" cross ScalatraCross exclude("com.typesafe.akka", "akka-actor")
-)
-
-libraryDependencies ++= Seq(
-  "com.novocode" % "junit-interface" % "0.10" % "test"
+  "ch.qos.logback" % "logback-core" % "1.1.2" % "test",
```
Comment (xvrl) [inline build.sbt]: any reason we need to add all the logging dependencies here?

Comment (gianm) [inline build.sbt]: the tests won't be able to log anything without them

File: build.sbt (hunk 1 above)
Comment (xvrl) [inline build.sbt]: just released 1.11.0, so feel free to update

Comment (gianm) [inline build.sbt]: ok, updated

**Pull request merged**